src/actions/interface.jl

```diff
@@ -6,5 +6,35 @@ entity based on the action on the given roadway.
 """
 propagate(veh::Entity{S,D,I}, action::A, roadway::R, Δt::Float64) where {S,D,I,A,R} = error("propagate not implemented for Entity{$S, $D, $I}, actions $A, and roadway $R")
 
+propagate(veh::Entity{S,D,I}, state::S, roadway::R, Δt::Float64) where {S,D,I,R} = state
 
-propagate(veh::Entity{S,D,I}, state::S, roadway::R, Δt::Float64) where {S,D,I,R}= state
\ No newline at end of file
+"""
+Mapping from actions to vehicle ids. The main use case is for keeping track of the action history
+in the same way as the scene history in the `simulate!` function.
+
+Initialize as
+
+    ActionMapping(a, id)
+
+where `a` is an action which can be used to propagate entities. `id` is the entity identifier.
+"""
+struct ActionMapping{A,I}
```

Comment:
is `ActionMapping` the best name for this? 
When I see mapping, I think dictionaries or functions. 
Maybe `EntityAction` or `AgentAction`?